Loading pipeline components...:   0%|          | 0/7 [00:00<?, ?it/s]/home/cn356/.conda/envs/diffuser/lib/python3.9/site-packages/transformers/models/clip/feature_extraction_clip.py:28: FutureWarning: The class CLIPFeatureExtractor is deprecated and will be removed in version 5 of Transformers. Please use CLIPImageProcessor instead.
  warnings.warn(
Loading pipeline components...:  29%|██▊       | 2/7 [00:00<00:02,  2.02it/s]Loading pipeline components...:  43%|████▎     | 3/7 [00:01<00:01,  2.86it/s]Loading pipeline components...:  57%|█████▋    | 4/7 [00:01<00:01,  2.94it/s]/home/cn356/.conda/envs/diffuser/lib/python3.9/site-packages/transformers/tokenization_utils_base.py:1601: FutureWarning: `clean_up_tokenization_spaces` was not set. It will be set to `True` by default. This behavior will be depracted in transformers v4.45, and will be then set to `False` by default. For more details check this issue: https://github.com/huggingface/transformers/issues/31884
  warnings.warn(
Loading pipeline components...:  86%|████████▌ | 6/7 [00:01<00:00,  4.16it/s]Loading pipeline components...: 100%|██████████| 7/7 [00:01<00:00,  3.98it/s]
  0%|          | 0/50 [00:00<?, ?it/s]  2%|▏         | 1/50 [00:00<00:09,  4.94it/s]  8%|▊         | 4/50 [00:00<00:03, 14.89it/s] 14%|█▍        | 7/50 [00:00<00:02, 19.91it/s] 20%|██        | 10/50 [00:00<00:01, 22.78it/s] 26%|██▌       | 13/50 [00:00<00:01, 24.52it/s] 32%|███▏      | 16/50 [00:00<00:01, 25.67it/s] 38%|███▊      | 19/50 [00:00<00:01, 26.40it/s] 44%|████▍     | 22/50 [00:00<00:01, 26.89it/s] 50%|█████     | 25/50 [00:01<00:00, 27.23it/s] 56%|█████▌    | 28/50 [00:01<00:00, 27.46it/s] 62%|██████▏   | 31/50 [00:01<00:00, 27.34it/s] 68%|██████▊   | 34/50 [00:01<00:00, 27.54it/s] 74%|███████▍  | 37/50 [00:01<00:00, 27.68it/s] 80%|████████  | 40/50 [00:01<00:00, 27.80it/s] 86%|████████▌ | 43/50 [00:01<00:00, 27.85it/s] 92%|█████████▏| 46/50 [00:01<00:00, 27.89it/s] 98%|█████████▊| 49/50 [00:01<00:00, 27.93it/s]100%|██████████| 50/50 [00:01<00:00, 25.57it/s]
Traceback (most recent call last):
  File "/home/cn356/diffusers/test.py", line 12, in <module>
    if not os.path.exists('./images'):
NameError: name 'os' is not defined
